lib/state/appStore.ts

```diff
@@ -124,15 +133,25 @@ export const useAppStore = create<AppState>()((set, get) => {
   const initialSelectedWalletId = +(
     secureStorage.getItem(selectedWalletIdKey) || "0"
   );
+
+  const isBiometricSupported = secureStorage.getItem(isBiometricSupportedKey) === "true";
```

Comment:
Why is if it is supported saved to our storage?

Reply:
We show an alert in the Settings > Security page if it is not supported

Reply:
Is there a problem with not storing these things to our appState at all and just requesting it where needed?

Reply:
Not actually, updated to remove `isBiometricSupported` from strorage ✅